### PR TITLE
Replace BeeSwarm with gobierto-vizzs

### PR DESCRIPTION
**Pull request by Crashillo**
## :v: What does this PR do?
Uses the new shared library instead the built-in chart. Purge files.

### CODE REVIEW

File: app/javascript/gobierto_visualizations/modules/contracts_controller.js [hunk 1]
```diff
@@ -11,11 +11,11 @@ import VueRouter from "vue-router";
 import { getRemoteData, calculateSumMeanMedian } from "../webapp/lib/utils";
 import { EventBus } from "../webapp/lib/mixins/event_bus";
 
-import { checkAndReportAccessibility } from "lib/vue/accesibility";
+// import { checkAndReportAccessibility } from "lib/vue/accesibility";
```
Comment (ferblape) [inline app/javascript/gobierto_visualizations/modules/contracts_controller.js]: why commenting this?

Comment (Crashillo) [reply app/javascript/gobierto_visualizations/modules/contracts_controller.js]: Accessibility creates a lot of noise in the console, I use to disable it meanwhile, but I forgot to restore it :(

Comment (Crashillo) [reply app/javascript/gobierto_visualizations/modules/contracts_controller.js]: Restored

**Comment by Crashillo** [comment]
BeeSwarm categories sorting included as well. This can be merged when green tests

**Review by jorgeatgu — APPROVED** [review]

**Pull request merged**